packages/framework/src/Component/String/TransformString.php

```diff
@@ -126,4 +126,13 @@ public static function replaceOccurences(string $search, string $replace, string
 
         return preg_replace($search, $replace, $subject, $limit);
     }
+
+    /**
+     * @param string $string
+     * @return string
+     */
+    public static function replaceNonPrintableCharactersByQuestionMark(string $string): string
```

Comment:
I think this method should have some more accurate name, as replacing non printable characters is not exactly what it does. Maybe replaceNonUTF8CharactersByQuestionMar

Reply:
Ok, replaced to `replaceInvalidUtf8CharactersByQuestionMark` as we agreed 👍